sessionctx/variable/session.go

```diff
@@ -380,6 +397,8 @@ func (tc *TransactionContext) GetCurrentSavepoint() TxnCtxNeedToRestore {
 	}
 	pessimisticLockCache := make(map[string][]byte, len(tc.pessimisticLockCache))
 	maps.Copy(pessimisticLockCache, tc.pessimisticLockCache)
+	CurrentStmtPessimisticLockCache := make(map[string][]byte, len(tc.CurrentStmtPessimisticLockCache))
+	maps.Copy(CurrentStmtPessimisticLockCache, tc.CurrentStmtPessimisticLockCache)
```

Comment:
The map `CurrentStmtPessimisticLockCache` is never used?  Should put it into `TxnCtxNeedToRestore`?

Reply:
Oh, I think no need to put `CurrentStmtPessimisticLockCache` into `TxnCtxNeedToRestore`, then we can just remove those 2 line code?

Reply:
> Oh, I think no need to put CurrentStmtPessimisticLockCache into TxnCtxNeedToRestore, then we can just remove those 2 line code?

Yes. It must be a mistake during refactoring.